src/Microsoft.AspNetCore.Mvc.ViewFeatures/RemoteAttribute.cs

```diff
@@ -2,46 +2,29 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations;
-using System.Globalization;
-using System.Linq;
-using Microsoft.AspNetCore.Mvc.DataAnnotations;
-using Microsoft.AspNetCore.Mvc.Internal;
 using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
 using Microsoft.AspNetCore.Mvc.Routing;
 using Microsoft.AspNetCore.Mvc.ViewFeatures;
-using Microsoft.AspNetCore.Routing;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Localization;
-using Microsoft.Extensions.Options;
 
 namespace Microsoft.AspNetCore.Mvc
 {
     /// <summary>
-    /// A <see cref="ValidationAttribute"/> which configures Unobtrusive validation to send an Ajax request to the
+    /// A <see cref="RemoteAttributeBase"/> for controllers which configures Unobtrusive validation to send an Ajax request to the
     /// web site. The invoked action should return JSON indicating whether the value is valid.
     /// </summary>
     /// <remarks>Does no server-side validation of the final form submission.</remarks>
     [AttributeUsage(AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
-    public class RemoteAttribute : ValidationAttribute, IClientModelValidator
+    public class RemoteAttribute : RemoteAttributeBase
     {
-        private string _additionalFields = string.Empty;
-        private string[] _additionalFieldsSplit = Array.Empty<string>();
-        private bool _checkedForLocalizer;
-        private IStringLocalizer _stringLocalizer;
-
         /// <summary>
         /// Initializes a new instance of the <see cref="RemoteAttribute"/> class.
         /// </summary>
         /// <remarks>
         /// Intended for subclasses that support URL generation with no route, action, or controller names.
         /// </remarks>
         protected RemoteAttribute()
```

Comment:
Should this constructor be annotated with obsolete now when RemoteAttributeBase exists?